src/coreclr/inc/corhdr.h

```diff
@@ -1136,12 +1136,9 @@ typedef struct IMAGE_COR_ILMETHOD_SECT_FAT
 typedef enum CorExceptionFlag                       // definitions for the Flags field below (for both big and small)
 {
     COR_ILEXCEPTION_CLAUSE_NONE,                    // This is a typed handler
-    COR_ILEXCEPTION_CLAUSE_OFFSETLEN = 0x0000,      // Deprecated
-    COR_ILEXCEPTION_CLAUSE_DEPRECATED = 0x0000,     // Deprecated
     COR_ILEXCEPTION_CLAUSE_FILTER  = 0x0001,        // If this bit is on, then this EH entry is for a filter
     COR_ILEXCEPTION_CLAUSE_FINALLY = 0x0002,        // This clause is a finally clause
     COR_ILEXCEPTION_CLAUSE_FAULT = 0x0004,          // Fault clause (finally that is called on exception only)
-    COR_ILEXCEPTION_CLAUSE_DUPLICATED = 0x0008,     // duplicated clause. This clause was duplicated to a funclet which was pulled out of line
 } CorExceptionFlag;
```

Comment:
```suggestion
    COR_ILEXCEPTION_CLAUSE_DUPLICATED = 0x0008,     // Deprecated: Duplicated clause. This clause was duplicated to a funclet which was pulled out of line
    COR_ILEXCEPTION_CLAUSE_SAMETRY    = 0x0010,     // This clause covers same try block as the previous one
} CorExceptionFlag;
```

Reply:
And also change the one place in VM that uses hardcoded 0x10 to use `COR_ILEXCEPTION_CLAUSE_SAMETRY`

Reply:
I think we want to keep `COR_ILEXCEPTION_CLAUSE_DUPLICATED` here even through it is deprecated since it was part of the file format.